### ISSUE TITLE
"Send Everything" fails when sending to contract. Tx fee is deducted from sending account but nothing else happens.

### ISSUE THREAD

**Issue by bamos01** [ethereum/mist#357]
The tx fee is deducted from the account, and tx appears to go through on mist and blockchain explorers but the account balances are not updated.

Here is tx hash:  0xa0b4a76091390e1ae3540b641bfdf446ef9c88881073aa292d130aa8a850db79

I had 1 ETH in the account.  I was attempting to send 0.99941512 to a contract wallet and pay 0.00058488 in tx fee (calculated by mist).  Although the contract appeared to be successful, all that happened was the tx fee was deducted from the sending account's balance.  I have replicated this error several times to confirm.  No errors reported on etherscan.io and tx appears valid there.

**Comment by bamos01** [comment]
update:  now etherscan.io is reporting an error during contract execution, but I can't figure out why there was an error.   I assume this is still something related to the fee not being high enough to cover the cost.

**Comment by bamos01** [comment]
confirmed out of gas exception.  Mist estimates gas /tx fee correctly for transaction to contract if balance is not being emptied of sending account, but it appears that gas is higher if transaction is emptying sending account.

**Comment by koeppelmann** [comment]
can confirm the same problem. Maybe the "send Everything" function should only be available for sending money to a none contract address because contracts do not necessarily have a upfront calculable gas cost and if you allow a higher gas limit you will always end up with some "dust" on the address you are trying to empty.

**Comment by dorroughdev** [comment]
WIndows 7 
Mist beta 7.2
Trying to "send all" eth fails and sends nothing and steals my TX fee

**Comment by jldinh** [comment]
Same problem.
It seems to be because the wallet sets the maximum fee (labelled "Provide maximum fee" in the "Send transaction" confirmation window) to 144,091 gas if you do not tick "Send everything", but to 44,091 if you do. If this is not enough, contract execution will fail.

**Comment by bamos01** [comment]
Right- thats because if you set the max gas, you cant empty the account
because it only needs 44091 if you sending to another account.  I dont
think there is a simple solution right now- but there should be a warning
against ticking 'send everything' if you are sending to a contract

**Comment by attila-lendvai** [comment]
same here. i've also transferred unwittingly some ether to the miners...

i'm trying to split my ETC/ETH, and i cannot find any discussion/advice online on how to empty the pre-split accounts. any hints?
